services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/switches/SwitchRulesSpec.groovy

```diff
@@ -519,9 +577,11 @@ class SwitchRulesSpec extends BaseSpecification {
         flowHelper.randomFlow(srcSwitch, dstSwitch)
     }
 
-    List<FlowEntry> getFlowRules(SwitchId switchId) {
+    List<FlowEntry> getRules(SwitchId switchId, defaultRules = true) {
```

Comment:
This will be conflicting with my patch about new bfd rule and should be resolved to my advantage, since default rules are different across actons, centecs and noviflows

Reply:
Fixed.